src/components/Homepage/index.js

```diff
@@ -46,88 +46,116 @@ const Homepage = () => {
   });
 
   return (
-    <Grid item container xs={12}>
+    <Grid
+      item
+      xs={12}
+      container
+      alignItems="space-between"
+      justifyContent="center"
+    >
       {page === 0 && (
-        <Grid item xs={12} className="Homepage_heading_Container">
+        <Grid
+          item
+          xs={12}
+          className="Homepage_heading_Container"
+          container
+          justifyContent="center"
+        >
```

Comment:
I'm not a fan of these 3rd party component like Grid. If you look closely, you'd notice that the styles are being maintained in 2 separate files. One in your css file and other here. Like `justifyContent` is being supplied as prop but if all the styles are in one place then it would be easy to maintain. You don't have to refactor everything now. Just want to let you know. If you are still confused then let me know